api/_examples/cloud-accouts/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/lacework/go-sdk/api"
+)
+
+func main() {
+	// TODO @afiune maybe think about a way to inject CI credentials and
+	// run these examples as part of our CI pipelines
+	lacework, err := api.NewClient(os.Getenv("LW_ACCOUNT"),
+		api.WithSubaccount(os.Getenv("LW_SUBACCOUNT")),
+		api.WithApiKeys(os.Getenv("LW_API_KEY"), os.Getenv("LW_API_SECRET")),
+		api.WithApiV2(),
+	)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	res, err := lacework.V2.CloudAccounts.List()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, account := range res.Data {
+		support := "Unsupported"
+		switch account.Type {
+		case api.AwsCtSqsCloudAccount.String():
+			//case api.AwsCfgCloudAccount:
+			//case api.GcpCfgCloudAccount:
+			//case api.GcpAtSesCloudAccount:
+			support = "Supported"
+		}
+
+		// Output: INTEGRATION-GUID:INTEGRATION-TYPE:[Supported|Unsupported]
+		fmt.Printf("%s:%s:%s\n", account.IntgGuid, account.Type, support)
+	}
+
+	awsCtSqsData := api.AwsCtSqsData{
+		Credentials: api.AwsCtSqsCredentials{
+			RoleArn:    "arn:aws:iam::123456789000:role/lw-iam-b8c91298",
```

Comment:
add a comment: WARNING: fake values, please insert your own ?

Reply:
This lives inside a directory named `_examples/`, I think every file here inherits that WARNING message, don't you think so?